package.json

```diff
@@ -86,5 +87,10 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "prettier": {
+    "printWidth": 120,
+    "semi": true,
+    "singleQuote": true
```

Comment:
Let's keep this out of this PR

Reply:
After merging in current master, this is no longer in the diff, since it was added in #303 .
https://github.com/lazd/coronadatascraper/pull/228/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2